### PR TITLE
Add libdb/5.3.28 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **libdb/5.3.28**

Depends on #852

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libdb/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+import shutil
+
+
+class LibdbConan(ConanFile):
+    name = "libdb"
+    description = "Berkeley DB is a family of embedded key-value database libraries providing scalable high-performance data management services to applications"
+    topics = ("conan", "gdbm", "dbm", "hash", "database")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.oracle.com/database/berkeley-db"
+    license = ("BSD", "LGPLv2", "Sleepycat")
```
Comment (uilianries) [inline recipes/libdb/all/conanfile.py]:
```suggestion
    license = ("BSD-3-Clause")
```

The license file only has a description which is BSD-3

Comment (madebr) [reply recipes/libdb/all/conanfile.py]: Thanks for noticing!

**Comment by madebr** [comment]
@uilianries It will fail again.
I think it is a problem with visual studio 14 not accepting more then one `.properties` file.
I do not know how to skip including the property file generated by `MSBuild`.

**Comment by uilianries** [comment]
@madebr Indeed. Is that file in the project? Can you remove it from source folder?

**Comment by madebr** [comment]
@uilianries 
It is generated by the `visual_studio` generated.
I use it to inject the paths of the include directories and libraries.

**Comment by madebr** [comment]
Woohoo!!!

Please review while it's hot :fire: :smile: 
@uilianries  @SSE4 @danimtb

**Pull request merged**